components/mixins/experience-event/industry-verticals/experienceevent-upsell-details.schema.json

```diff
@@ -30,6 +30,11 @@
         "xdm:upsell": {
           "type": "object",
           "properties": {
+            "xdm:upsellSteps": {
+              "title": "Upsell Steps",
+              "$ref": "https://ns.adobe.com/xdm/datatypes/transaction",
```

Comment:
Should this be "tool-usage" ? majority of step properties are defined as "tool-usage"

Reply:
You are right.  I just pushed a fix.